src/extension/index.ts

```diff
@@ -0,0 +1,20 @@
+import * as ProxyBindings from "./proxyBindings";
+
+export type ProxyAPI = typeof ProxyBindings;
+
+/**
+ * Initialize the window instance with bindings to proxy API functions
+ *
+ * @export
+ * @param { ProxyAPI } globalOrWindow
+ */
+export function bindToWindow(
+  globalOrWindow: typeof globalThis | Window,
```

Comment:
might as well have the global or window test here instead of passing it to the method (also isn't globalThis technically global or window ? Though that was the point of globalThis)

Reply:
Learned something new today. Thought `globalThis` was on related to the node.js, read the MDN docs on it.